system_tests/language.py

```diff
@@ -115,3 +115,10 @@ def test_analyze_entities_from_blob(self):
         document = Config.CLIENT.document_from_url(gcs_url)
         entities = document.analyze_entities()
         self._check_analyze_entities_result(entities)
+
+    def test_analyze_sentiment(self):
+        positive_msg = 'Jogging is fun'
+        document = Config.CLIENT.document_from_text(positive_msg)
+        sentiment = document.analyze_sentiment()
+        self.assertEqual(sentiment.polarity, 1)
+        self.assertTrue(0.5 < sentiment.magnitude < 1.5)
```

Comment:
And that rhymes with "Jogging is fun". The value was 0.6, so maybe I should've done `0 < val < 1`. Either way, I think this will need some tweaking as the backend matures.

Any suggestions for what to do in the meantime?